packages/map/theme/lumo/vaadin-map-styles.js

```diff
@@ -0,0 +1,150 @@
+/**
```

Comment:
Found an issue with the [drag to zoom interaction](https://openlayers.org/en/latest/apidoc/module-ol_interaction_DragZoom-DragZoom.html), which is configured by default (press Shift, then drag rectangle). During the interaction a rectangle should show up, indicating the area that is to be zoomed in. With our styles nothing shows up.

Reply:
Ah, right, that’s the control I forgot about. I did notice that in the examples and docs, but pushed it aside until everything else was done.

Reply:
Also, the zoom slider is another thing I forgot about.

Reply:
Added these styles, drag-to-zoom and zoom slider. "Mouse position" was also only half-done, that’s also fixed now.